### DISCUSSION TITLE
Option to preserve white space?

### DISCUSSION THREAD

**Discussion by samgormancirdan** [jhy/jsoup#2203] [Ideas]
JSoup tidies the HTML and strips out extra whitespace from the text. While this is nice we need to preserve the white space. I have extended the code to ass a preserveWhiteSpace option to the doc.outputSettings and added the relevant unit tests. If you would like it I'd be happy to send the changes.

**Comment by jhy** [comment]
If you disable pretty-print in the output settings, it also preserves the whitespace.
